tf2_ros/include/tf2_ros/static_transform_broadcaster.h

```diff
@@ -43,29 +43,23 @@
 namespace tf2_ros
 {
 
-
-/** \brief This class provides an easy way to publish coordinate frame transform information.  
- * It will handle all the messaging and stuffing of messages.  And the function prototypes lay out all the 
+/** \brief This class provides an easy way to publish coordinate frame transform information.
+ * It will handle all the messaging and stuffing of messages.  And the function prototypes lay out all the
  * necessary data needed for each message.  */
 
 class StaticTransformBroadcaster{
 public:
-
-  TF2_ROS_PUBLIC
-  StaticTransformBroadcaster(rclcpp::Node::SharedPtr node);
-
-  template<class AllocatorT = std::allocator<void>>
+  /** \brief Node interface constructor */
+  template<class NodeT, class AllocatorT = std::allocator<void>>
   StaticTransformBroadcaster(
-    rclcpp::node_interfaces::NodeTopicsInterface::SharedPtr node_topics_interface)
+    NodeT && node,
+    const rclcpp::QoS & qos = rclcpp::QoS(100),
+    const rclcpp::PublisherOptionsWithAllocator<AllocatorT> & options =
+      rclcpp::PublisherOptionsWithAllocator<AllocatorT>())
   {
-    auto qos = rclcpp::QoS(rclcpp::KeepLast(100));
-    // TODO(tfoote) latched equivalent
```

Comment:
Make it latched or leave the TODO. @tfoote do you have any input?

Reply:
just for completeness: `rclcpp::QoS(rclcpp::KeepLast(100))` is not the same as `rclcpp::QoS(100)`? Should be default to keep last then?

Reply:
It is the same.

Reply:
Also, at least restore the TODO until @tfoote can respond.